sdk/src/wallet/operations/transaction/account.rs

```diff
@@ -81,3 +103,9 @@ where
             .await
     }
 }
+
+#[derive(From)]
+pub enum BlockIssuerKeySource {
+    Key(PublicKey),
+    Path(Bip44),
```

Comment:
I'd maybe call the variant BipPath, Path is a bit too generic IMO

Reply:
alrighty